.releaserc

```diff
@@ -0,0 +1,31 @@
+{
+    "branches": [
+        "master",
```

Comment:
This is the branch that releases the new version of the package right? Let's change this to `production`.